src/GuiBase/BaseApplication.hpp

```diff
@@ -68,7 +69,7 @@ class RA_GUIBASE_API BaseApplication : public QApplication
                      const WindowFactory& factory,
                      QString applicationName  = "RadiumEngine",
                      QString organizationName = "STORM-IRIT" );
-    virtual ~BaseApplication();
+    ~BaseApplication() override;
```

Comment:
According to Cpp guidelines C.128: (https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rh-override)
> If a base class destructor is declared virtual, one should avoid declaring derived class destructors virtual or override.
This is because destructors of a derived class does not override of the base destructor but is used in a chain call.

Reply:
That was a clang tidy suggestion, I revert back.